packages/pangea-sdk/pangea/asyncio/services/intel.py

```diff
@@ -864,6 +865,7 @@ async def user_breached_bulk(
             usernames (List[str]): An username's list to search for
             ips (List[str]): An ip's list to search for
             phone_numbers (List[str]): A phone number's list to search for. minLength: 7, maxLength: 15.
+            domains (List[str]): An domain's list to search for
```

Comment:
just copying this from the docs
```suggestion
            domains (List[str]): Search for user under these domains.
```